### ISSUE TITLE
Midpoint Review - Accessibility Feedback - [AuthExp, My VA 3.0]

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#139239]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** To provide semantic structure to assistive technology, when you use cards, they should be structured using list (`<li>`), so that users of assistive technology (i.e. screen readers) can understand when a card begins/ends. This will mimic the sighted experience and enhances the experience for screen reader users.
https://designsystem.digital.gov/components/card/accessibility-tests/#screen-reader-announces-groups-of-cards-as-an-unordered-list
<img width="537" height="203" alt="cards" src="https://github.com/user-attachments/assets/d406b4b1-ccda-471a-be6c-348c8599851c" />

- [ ] 

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app
